concourse/tasks/remove-healthcheck-db.yml

```diff
@@ -13,11 +13,17 @@ run:
     - |
       ./paas-cf/concourse/scripts/import_bosh_ca.sh
       . ./config/config.sh
-      if curl -I -f $API_ENDPOINT/info; then
-        echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
-        cf target -o testers -s healthcheck
-        cf delete healthcheck -f -r
-        if cf services | grep -q healthcheck-db; then
-          cf delete-service healthcheck-db -f
-        fi
+      if ! curl -I -f $API_ENDPOINT/info; then
+        echo "CF API unavailable. Skipping..."
+        exit 0
+      fi
+      echo | cf login -a ${API_ENDPOINT} -u ${CF_ADMIN} -p ${CF_PASS}
+      if ! cf org testers > /dev/null; then
```

Comment:
That would work, but I think it's better as it is now for a couple of reasons:

* I had to think for a bit about how the pipe in the command would interact with the conditional, which means other reading this are likely to have to as well. I think it's clearer what's going on at the moment.

* It conflates login failure with the API being unavailable. We want to skip this if the API isn't there (because the deployment doesn't exist), but not if authentication fails for another reason (eg wrong password etc).